config/os_fetcher.go

```diff
@@ -53,3 +53,11 @@ func (o *OsFetcher) Get(key string) (val string, ok bool) {
 
 	return v, ok
 }
+
+func (o *OsFetcher) All() map[string]string {
+	return nil
+}
+
+func (o *OsFetcher) Set(key, value string) {}
```

Comment:
If the `Set` and `Del` methods remain on this type, I think a `panic` would be appropriate here, since these methods explicitly do nothing which does not agree with the caller's expectations.